google_api_lib/utils.py

```diff
@@ -7,6 +8,12 @@
 from celery import Task
 
 
+class Priority(Enum):
```

Comment:
Should we move this somewhere else? Admittedly not really sure where.

Reply:
moved to settings.py